pontoon/base/views.py

```diff
@@ -104,6 +105,17 @@ def locale(request, locale, template='locale.html'):
     return render(request, template, data)
 
 
+def locales(request, template='locales.html'):
```

Comment:
Actually no - we already use `locale` for single team (`/sl`), so we should use `locales` (or `teams`) for teams overview. The same pattern as we use with `project` and `projects`.

And I agree about the template, this is a relic from the past.